packages/arb-token-bridge-ui/src/components/TransferPanel/TokenSearch.tsx

```diff
@@ -186,6 +221,21 @@ function TokenRow({ style, onClick, token }: TokenRowProps): JSX.Element {
               {tokenSymbol}
             </span>
             <span className="text-xs text-gray-500">{tokenName}</span>
+
+            {isArbitrumToken && (
+              <Tooltip content={arbitrumTokenTooltipContent}>
+                <StatusBadge variant="green">
+                  <CheckCircleIcon className="h-4 w-4" />
+                  <span className="text-xs">Official token</span>
```

Comment:
Suggestion to change the badge to `Official ARB Token` because we wouldn't want to diminish the authenticity of other tokens by calling them not official. They're equally legit in their own chains/utilities.

<img width="303" alt="image" src="https://user-images.githubusercontent.com/7558499/226871733-733fa888-0be7-4017-a69f-803ff2f465e2.png">

Reply:
Done